baseapp/abci.go

```diff
@@ -130,24 +131,6 @@ func (app *BaseApp) Info(ctx context.Context, req *abci.RequestInfo) (*abci.Resp
 	}, nil
 }
 
-// FilterPeerByAddrPort filters peers by address/port.
```

Comment:
need to keep these two

Reply:
As above, this was moved to `baseapp.p2p.go`